src/main/java/com/offdk/play/model/game/MatchPlayer.java

```diff
@@ -7,28 +7,29 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.MoreObjects;
+import com.offdk.play.model.slack.User;
 import com.offdk.play.rating.RatingType;
 
 import io.vavr.control.Try;
 
 public class MatchPlayer {
-  private final String id;
+  private final User user;
   private final MatchPlayerType type;
   private Optional<Integer> score;
   private Optional<BigDecimal> rating;
 
   // Needed to allow spring-data to serialize data to object
   @PersistenceConstructor
-  MatchPlayer(String id, MatchPlayerType type, Optional<Integer> score, Optional<BigDecimal> rating) {
-    this.id = id;
+  MatchPlayer(User user, MatchPlayerType type, Optional<Integer> score, Optional<BigDecimal> rating) {
+    this.user = user;
     this.type = type;
     this.score = score;
     this.rating = rating;
   }
 
   @VisibleForTesting
```

Comment:
Not sure how allowing multiple ratings would impact MatchPlayer, if you want the ratings to live on Player.

But if you do want this break, no reason to leave this constructor around anymore. It's just cluttering the code and the test cases should probably be fixed to reflect that change.